### PR TITLE
Added gmail functionality

### PR DESCRIPTION
**Pull request by Likhovodov**
This should allow us to send emails using gmail's smtp server.

How you can help: Pull it in and test it out. Make sure emails are able to be sent. I tested it, and it works for me. Watch out though, stuff could land in your spam inbox.

### CODE REVIEW

File: simcon_project/simcon_project/settings.py [hunk 1]
```diff
@@ -130,9 +125,18 @@
 
 USE_TZ = True
 
+# Email
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_USE_TLS = True
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'simcon.dev@gmail.com'
+EMAIL_HOST_PASSWORD = 'zhtsjuriiabkdahg'
```
Comment (rosasck) [inline simcon_project/simcon_project/settings.py:134]: aaaaaAAAAAAAHHH NOOOOOOOOOOOOOO

**Review by rosasck — APPROVED** [review]

File: simcon_project/simcon_project/settings.py (hunk 1 above)
Comment (rosasck) [reply simcon_project/simcon_project/settings.py:134]: although first email goes to spam, the rest go to inbox

Comment (Likhovodov) [reply simcon_project/simcon_project/settings.py:134]: What's up?

**Review by ericharper1 — APPROVED** [review]
Awesome! Registration link is nice and compact too

**Review by Mkay971 — APPROVED** [review]
looks good!

**Pull request merged**